corpora/csip/metadata/metshdr/CSIP8/mets-xml_metsHdr_LASTMODDATE_in_future/METS.xml

```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!-- Minimal IP with one representation -->
+<!-- This file is designed to be compliant with CSIP v.2.0.3, 08.01.2020, https://earkcsip.dilcis.eu/ -->
+<!-- Note: the values of ID attributes are chosen to ease understanding, but they could be anything that satisfies the requirements of xml:ID and CSIP. -->
+
+<!-- CSIP1 mets/@OBJID - Package Identifier - [1..1] -->
+<!-- CSIP2 mets/@TYPE - Content Category - [1..1] -->
+<!-- CSIP6 mets/@PROFILE - METS Profile - [1..1] -->
+<!-- Section 5.3.1. - mets root element MUST define all relevant namespaces and locations of XML schemas using the @xmlns and @xsi:schemaLocation attributes. -->
+<mets 
+  xmlns="http://www.loc.gov/METS/" 
+  xmlns:csip="https://DILCIS.eu/XML/METS/CSIPExtensionMETS" 
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+  xmlns:xlink="http://www.w3.org/1999/xlink" 
+  xsi:schemaLocation="
+    http://www.loc.gov/METS/ https://www.loc.gov/standards/mets/mets.xsd
+    https://DILCIS.eu/XML/METS/CSIPExtensionMETS http://earkcsip.dilcis.eu/schema/DILCISExtensionMETS.xsd
+    http://www.w3.org/1999/xlink http://www.loc.gov/standards/xlink/xlink.xsd"
+  OBJID="minimal_IP_with_1_representation" 
+  TYPE="Mixed"  
+  PROFILE="https://earkcsip.dilcis.eu/profile/E-ARK-CSIP.xml">
+
+  <!-- CSIP117 mets/metsHdr - Package header - [1..1] -->
+  <!-- CSIP7   mets/metsHdr/@CREATEDATE - Package creation date - [1..1] -->
+  <!-- CSIP8   mets/metsHdr/@LASTMODDATE - Package last modification date - [0..1] -->
+  <!-- CSIP9   mets/metsHdr/@csip:OAISPACKAGETYPE - OAIS Package type information - [1..1] -->
+  <metsHdr CREATEDATE="2019-04-14T20:00:00" csip:OAISPACKAGETYPE="SIP">
```

Comment:
I don't see a LASTMODDATE?

Reply:
Sorry, I forgot to say that the focus currently was only on testCase.xml, I didn't even notice CSIP8 had packages done. Ülle created new packages based on the valid package `valid_IP_with_SHOULD_MAY_1_rep` and these handle LASTMODDATE correctly.

In sum, let's only fix testCase.xml now, then I merge the PR, then Ülle uploads the correct packages and creates a new PR.